linter/rules/dimension_group_name_ends_with_date_time_or_day_pt.py

```diff
@@ -0,0 +1,10 @@
+from linter.rule import Rule
+from typing import Any, Tuple
+
+
+class DimensionGroupNameEndsWithDateTimeOrDayPt(Rule):
+    def applies_to() -> Tuple[str, ...]:
+        return ('dimension_group',)
+
+    def run(self, dimension_group: Any) -> bool:
+        return not dimension_group.get('name').endswith(('_date', '_time', '_day_pt'))
```

Comment:
Just curious, what is **day_pt** supposed to represent?

Reply:
It's how their dates are labeled in BigQuery. Developers carry that naming convention to Looker which they want to stop

Reply:
Gotcha.  This looks like another negative check class, so we should probably rename it to `should_not_end...`, but also **day_pt** is getting into more DIscord-domain.  I would keep this class here for _date and _time because those are more generic and relevant to other clients, and then create a new class called:

```
DimensionGroupNameShouldNotEndWith
```

or something similar, and pass a param set in the config file:

```
- rule: dimension_group_should_not_end_with
  severity: warning
  param_sets:
    - search_terms:
       - _day_pt
```

Reply:
Thanks for the feedback! I split the class in question into two classes. One to check for dimension groups ending with `_date` and `_time` and another which allows end users to define their search term via the param set.